code/datums/tgs_event_handler.dm

```diff
@@ -0,0 +1,41 @@
+/datum/tgs_event_handler/impl
+	var/datum/timedevent/reattach_timer
+
+/datum/tgs_event_handler/impl/HandleEvent(event_code, ...)
+	switch(event_code)
+		if(TGS_EVENT_REBOOT_MODE_CHANGE)
+			var/list/reboot_mode_lookup = list ("[TGS_REBOOT_MODE_NORMAL]" = "be normal", "[TGS_REBOOT_MODE_SHUTDOWN]" = "shutdown the server", "[TGS_REBOOT_MODE_RESTART]" = "hard restart the server")
+			var/old_reboot_mode = args[2]
+			var/new_reboot_mode = args[3]
+			message_admins("TGS: Reboot will no longer [reboot_mode_lookup["[old_reboot_mode]"]], it will instead [reboot_mode_lookup["[new_reboot_mode]"]]")
```

Comment:
```suggestion
			message_admins("TGS: Rebooting will no longer [reboot_mode_lookup["[old_reboot_mode]"]], it will [reboot_mode_lookup["[new_reboot_mode]"]] instead")
```

Reply:
nitpick but okay xD